src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java

```diff
@@ -607,6 +705,8 @@ public ActivityProfileState isActive(final AlignmentContext context, final Refer
      * @return List of variants discovered in the region (may be empty)
      */
     public List<VariantContext> callRegion(final AssemblyRegion region, final FeatureContext features, final ReferenceContext referenceContext) {
+        final HaplotypeCallerGenotypingEngine localEngine = getLocalGenotypingEngine(region);
```

Comment:
localEngine -> localGenotypingEngine (x the several times this appears below)

Reply:
Done, good call